docpages/example_programs/interactions_and_components/slashcommands-menu.md

```diff
@@ -0,0 +1,12 @@
+\page slashcommands-menu Slash commands
+
+Here you will find examples on how to use slash commands, subcommands, and different types or parameters.
+
+* \subpage slashcommands "Using Slash Commands"
+* \subpage clearing_slashcommands
+* \subpage subcommands "Slash command sub-commands"
+* \subpage application-command-autocomplete "Slash command auto completion"
+* \subpage discord-application-command-file-upload "Using file parameters in slash commands"
+* \subpage commandhandler "Unified message/slash command handler"
+
+\note Keep in mind, someone using a slash command is <b>NOT A MESSAGE!!</b> It is not a string but an interaction structure with its own data, and \ref resolved-objects "command parameters are objects".
```

Comment:
someone using a slash command?
Do you mean: `Keep in mind, a slash command when sent to your bot is <b>not sent as a message</b>! ...`